### ISSUE TITLE
lipo with "-info" / "-archs" result in "invalid magic number"

### ISSUE THREAD

**Issue by JCash** [konoui/lipo#10]
Hi, and thanks for this great project!

While I suspect it's not as common that the `-archs`/`-info` commands are used, I think it's probably good to list it as a difference in functionality from the standard lipo tool.

Currently, when invoking it it produces an error:
```
$ ./lipo -archs ./mylib.a
Error: can't figure out the architecture type of: ./mylib.a: invalid magic number

# original lipo
$ lipo -archs ./mylib.a
arm64
```

**Comment by konoui** [comment]
Thank you for your report!
We needs to handle [ar file format](https://en.wikipedia.org/wiki/Ar_(Unix)).
I will address it after investigating the format.

**Comment by konoui** [comment]
addressed the issue in version [v0.9.0](https://github.com/konoui/lipo/releases/tag/v0.9.0)
